gdal_subsetter/transform.py

```diff
@@ -889,14 +933,13 @@ def unpack_zipfile(self, zipfilename, output_dir, variables=None):
 
         tmpnc = None
         filelist_nc = get_files_from_unzipfiles(f'{output_dir}/unzip', 'nc')
-        is_nc = bool(filelist_nc)
-        if filelist_nc:
+        if len(filelist_nc) > 0:
             tmpnc = filelist_nc
             msg_nc = 'OK'
         else:
             msg_nc = 'no available data for the variables, not process.'
 
-        return is_tif, tmptif, msg_tif, is_nc, tmpnc, msg_nc
+        return tmptif, msg_tif, tmpnc, msg_nc
```

Comment:
`is_tif` and `is_nc` seem redundant where you can just check `len(tmptif) > 0` or `len(tmpnc) > 0`.